exercises/ansible_engine/3-variables/README.md

```diff
@@ -53,31 +53,29 @@ Add a play definition and some variables to your playbook.  These include addtio
     httpd_packages:
       - httpd
       - mod_wsgi
-    apache_test_message: This is a test message
+    apache_test_message: This is a test message from {{ ansible_hostname }}
     apache_max_keep_alive_requests: 115
 ```
-
+Variables and variables! In the above the vars label helps us create a user-defined list of variables that we can use throughout the playbook. There are also global variables that are defined by ansible itself. The example above is `ansible_hostname`. This will resolve to the current ansible host where the playbook is executing at runtime. The `{{` and `}}` is a variable templating notation used to indicate that we would like the variable replaced with its value. More on this in a minute... 
 
 ### Step 3:
 
 Add a new task called *install httpd packages*.
 
 ```yml
-{% raw %}
```

Comment:
Why do you remove the raw code snippets? Afaik we need them for the export to github pages.
For more information please see: https://github.com/jekyll/jekyll/issues/6217